### PR TITLE
feat(scan): happy-path NH ballot scanning

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview

This does add adjudication data to NH ballot interpretations, but since there are no VX ballot layouts for NH ballots they cannot be adjudicated yet. For now this will support the happy path only.

This PR also bumps the NH mark thresholds up to 10% marginal and 14% definite to account for differences in print scaling with the samples I have been working with. We may be able to bring them back down, but not knowing how these ballots will be printed might mean we need them to be higher.

## Demo Video or Screenshot

https://user-images.githubusercontent.com/1938/160691138-c6f98c0a-e64b-4ded-9773-f84b810addab.mov

Here's the interpreted votes for the ballot scanned in the video:

### Front
```json
{
  "President-and-Vice-President-of-the-United-States-18d1a55a": [
    {
      "id": "Joseph-R-Biden-and-Kamala-D-Harris-346756c0",
      "name": "Joseph R. Biden  and Kamala D. Harris",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "Governor-061a401b": [
    {
      "id": "Dan-Feltes-3fadf798",
      "name": "Dan Feltes",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "United-States-Senator-d3f1c75b": [
    {
      "id": "Jeanne-Shaheen-6e19622d",
      "name": "Jeanne Shaheen",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "Representative-in-Congress-24683b44": [
    {
      "id": "Ann-McLane-Kuster-67fdf060",
      "name": "Ann McLane Kuster",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "Executive-Councilor-bb22557f": [
    {
      "id": "Debora-B-Pignatelli-8c4cc1d3",
      "name": "Debora B. Pignatelli",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "State-Senator-391381f8": [
    {
      "id": "Nancy-Hendricks-d330559d",
      "name": "Nancy Hendricks",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "State-Representatives-7d3a8821": [
    {
      "id": "Brett-Gagnon-7a8c9838",
      "name": "Brett Gagnon",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "David-Hennessey-f1343f4f",
      "name": "David Hennessey",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Beatriz-Jauregui-11b83653",
      "name": "Beatriz Jauregui",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Steven-Katsos-077584b8",
      "name": "Steven Katsos",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Harold-Lynde-a54df7d7",
      "name": "Harold Lynde",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Lana-Paliy-595b276a",
      "name": "Lana Paliy",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Robert-S-Sherman-f10f4a3f",
      "name": "Robert S. Sherman",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Alejandro-Urrutia-1b574b86",
      "name": "Alejandro Urrutia",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Timothy-Wyatt-0105b8de",
      "name": "Timothy Wyatt",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Barbara-A-Blue-14608463",
      "name": "Barbara A. Blue",
      "partyId": "Democratic-aea20adb"
    },
    {
      "id": "Nancy-S-Brucker-4471faf7",
      "name": "Nancy S. Brucker",
      "partyId": "Democratic-aea20adb"
    }
  ]
}
```

### Back
```json
{
  "Sheriff-4243fe0b": [
    {
      "id": "Bill-Barry-0e677ca6",
      "name": "Bill Barry",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "County-Attorney-133f910f": [
    {
      "id": "Michael-Conlon-1a61fa16",
      "name": "Michael Conlon",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "County-Treasurer-87d25a31": [
    {
      "id": "William-Bryk-eaf936ec",
      "name": "William Bryk",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "Register-of-Deeds-a1278df2": [
    {
      "id": "Mary-Ann-Crowell-e1ff59b9",
      "name": "Mary Ann Crowell",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "Register-of-Probate-a4117da8": [
    {
      "id": "Elizabeth-Ropp-4035eed1",
      "name": "Elizabeth Ropp",
      "partyId": "Democratic-aea20adb"
    }
  ],
  "County-Commissioner-d6feed25": [
    {
      "id": "Daniel-C-Hansberry-8c0291ec",
      "name": "Daniel C. Hansberry",
      "partyId": "Democratic-aea20adb"
    }
  ]
}
```

Note that since I just voted straight down the column, all the votes are for Democratic Party candidates.

## Testing Plan 
Tested manually with a properly-marked ballot, as shown in the video.

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/ballot-interpreter-nh/src/interpret/convert_marks_to_adjudication_info.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          (m) =>
+            m.gridPosition.contestId === option.contestId &&
+            ((m.gridPosition.type === 'option' &&
+              m.gridPosition.optionId === option.id) ||
+              (m.gridPosition.type === 'write-in' &&
+                m.gridPosition.writeInIndex ===
+                  option.optionIndex -
+                    (contest.type === 'candidate'
+                      ? contest.candidates.length
+                      : 0)))
```
Comment (jonahkagan) [inline libs/ballot-interpreter-nh/src/interpret/convert_marks_to_adjudication_info.ts:45]: any way to make this more readable? maybe pull out a few variables?

**Review by jonahkagan — APPROVED** [review]
Looks good! And good job adding some new test coverage

**Comment by eventualbuddha** [comment]
> Video didn't work for me FYI

Re-encoded the video with greater compatibility. Try again?

**Comment by jonahkagan** [comment]
Video works! Very satisfying to watch

**Pull request merged**